R/Spectra.R

```diff
@@ -1669,6 +1687,16 @@ setMethod("c", "Spectra", function(x, ...) {
     .concatenate_spectra(unname(list(unname(x), ...)))
 })
 
+setClassUnion("dataframeOrDataFrame", c("data.frame", "DataFrame"))
```

Comment:
why are you defining here this class union? in `MsBackend` you have already `data.frame`, `DataFrame` and `matrix` - I would re-use that actually.